client/elements/text/sc-simple-text.html

```diff
@@ -297,17 +333,30 @@ <h3>Could not find the sutta text.</h3>
               if (textElement) {
                   visible ? textElement.classList.add('infomode') : textElement.classList.remove('infomode');
                   for (var key in this.classTitles) {
```

Comment:
use `let` or `const` instead of `var`
https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75